app/main.py

```diff
@@ -1,3 +1,45 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> Distance:
```

Comment:
Do not forget to indicate all possible data types in the type annotation, so that `km` could be `float` as well